pageserver/src/tenant/mgr.rs

```diff
@@ -644,7 +844,12 @@ pub async fn load_tenant(
             remote_storage,
             deletion_queue_client
         };
-        let new_tenant = schedule_local_tenant_processing(conf, tenant_id, &tenant_path, generation, resources, None,  &TENANTS, ctx)
+
+        let mut location_conf = Tenant::load_tenant_config(conf, &tenant_id).ok().unwrap_or(LocationConf::default());
```

Comment:
`load_tenant_config` fills in a `LocationConf::default` on its own if the tenant conf is missing on disk.
What you're doing here is fill in default tenant config on, say, IO error.
Not good.

Reply:
Oops, you are quite right that this is a bug.  Fixed in 0f1cad1e26ac6ca699ffcce3fa54e48be81bab80